content/en/docs/contribute/style/style-guide.md

```diff
@@ -113,7 +115,7 @@ The copy is called a "fork". | The copy is called a "fork."
 
 ## Inline code formatting
 
-### Use code style for inline code, commands, and API objects
+### Use code style for inline code, commands, and API objects {#code-style-inline-code}
 
```

Comment:
You could update line 123 so that the table caption matches the section heading from line 118:
```{{< table caption = "Do and Don't - Use code style for inline code and commands and API objects" >}}```

Reply:
will do